mne/beamformer/_rap_music.py

```diff
@@ -156,7 +156,7 @@ def _make_dipoles(times, poss, oris, sol, gof):
     dipoles : list
         The list of Dipole instances.
     """
-    amplitude = sol * 1e9
+    amplitude = sol
```

Comment:
Why are you keeping amplitude, and not just using sol in Dipole(). Saves a few lines. 

In my opinion this is a bugfix not a feature change, as e.g. saving the dipoles created with rap music would multiplicate twice by 1e9.